src/Test/Test.php

```diff
@@ -35,19 +35,28 @@ class Test implements TestInterface
     private $variants;
 
     /**
+     * Case specific options
+     *
+     * @var array
+     */
+    private $options = [];
+
+    /**
      * Initializes a new instance of this class.
      *
      * @param string $identifier The identifier
      * @param VariantInterface[] $variants The variants that this test has.
+     * @param array $options Case specific test options.
      */
-    public function __construct($identifier, $variants = [])
+    public function __construct($identifier, $variants = [], array $options = [])
```

Comment:
I was thinking this too when we talked on Gitter. There are also options in the test bag. Which one should be used? Maybe @psren can review if this is correct?